sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -31,6 +30,7 @@
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.params.provider.Arguments;
+import org.opentest4j.TestAbortedException;
```

Comment:
Is there a usecase for bringing this exception in rather than using 'UnsupportedOperationException'?

Reply:
The JUnit special handle this exception, such that the tests get skipped rather than failing. This is the same exception that _assumeTrue_ (assertions that we already use today in few places) throws when assumption is false.

Reply:
The JUnit special handle this exception, such that the tests get skipped rather than failing. This is the same exception that assumeTrue (assertions that we already use today in few places) throws when assumption is false.

Reply:
Since this isn't a special case and we do something similar like it in other tests, I would expect similar behaviour (i.e. using what is currently done). I don't have strong opinions but if we're going to use TestAbortedException for this behaviour, we should update our other messaging tests to do the same so people know what to expect; or keep using assumeTrue.

Reply:
Sounds good, will update to throw this exception via _assumeTrue_, so we're consistent.